NAMESPACE

```diff
@@ -18,12 +18,9 @@ export(.diff)
 export(.unvf)
 export(.messages)
 export(.quietDuringUnitTest)
-export(.dataFrameToRowList)
```

Comment:
I'm guessing we can also get rid of:
- `.newFootnotes`
- `.addFootnote`
- `.diff`
- `.quietDuringUnitTest`

Reply:
And either we should move the classical meta analysis functions (like you suggested, which is a good idea) or they should be added here

Reply:
done